src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class GenerateEncodingTable : Task
+    {
+        private const string CommentIndicator = "#";
+        private const char FieldDelimiter = ';';
+
+        [Required]
+        public string IANAMappings { get; set; }
+
+        [Required]
+        public string PreferedIANANames { get; set; }
+
+        [Required]
+        public string OutputDataTable { get; set; }
+
+        public string Namespace { get; set; }
+        public string ClassName { get; set; }
```

Comment:
... I don't think there is one?  There isn't one in the same namespace (which is what I would expect).  Unless you mean from somewhere else?  
At least one existing task doesn't mark the optional attributes at all (well, I assumed it was optional).

I guess I could comment it.